solutions/usaco-893.mdx

```diff
@@ -0,0 +1,123 @@
+---
+id: usaco-893
+source: USACO Bronze 2019 January
+title: Guess the Animal
+author: Brad Ma
+---
+
+[Official Analysis (C++)](https://codeforces.com/blog/entry/22712)
```

Comment:
link to usaco one (?)

Reply:
oh yeah good catch i copy pasted a template from a codeforces problem and forgot to change